Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -620,11 +627,25 @@ - (BranchLinkProperties *)getFirstReferringBranchLinkProperties {
 }
 
 - (NSDictionary *)getFirstReferringParams {
-    return [BNCEncodingUtils decodeJsonStringToDictionary:self.preferenceHelper.installParams];
+    NSDictionary *origInstallParams = [BNCEncodingUtils decodeJsonStringToDictionary:self.preferenceHelper.installParams];
```

Comment:
That's not correct.  We always need to return the decoded original params.  The if block in the middle just adds more debug params if there are any.